subworkflows/MODEL_TO_DATA.nf

```diff
@@ -43,4 +44,5 @@ workflow MODEL_TO_DATA {
     SCORE(VALIDATE.output, UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE.output, ANNOTATE_SUBMISSION_AFTER_VALIDATE.output, params.scoring_script)
     UPDATE_SUBMISSION_STATUS_AFTER_SCORE(SCORE.output.map { tuple(it[0], it[2]) })
     ANNOTATE_SUBMISSION_AFTER_SCORE(SCORE.output)
+    SEND_EMAIL(params.view_id, image_ch.map { it[0] }, ANNOTATE_SUBMISSION_AFTER_SCORE.output)
```

Comment:
Since this step is informing submitters of a validation status, I think it might make more sense for this to occur right after `VALIDATE`. You also might want to go ahead and import this as `SEND_VALIDATION_EMAIL` or something like that to be clearer of what it is doing and to allow for us to send more emails in other steps.

Reply:
Or maybe I am misunderstanding and we are just going to send one email at the end that tells the submitter all of the results

Reply:
Do we want to send multiple e-mails per step, or 1 single e-mail summarizing everything after 1 run, to avoid spam?

Reply:
Yeah, I think you are right, sorry for the confusion!